source/index.js

```diff
@@ -40,7 +41,7 @@ const pushContent = {
   RENOTIFY: false,
```

Comment:
Chances are that the push badge icon will brake due to the path updates. Could we switch to absolute path as in the other cases?